lib/sensors/brakes.cpp

```diff
@@ -0,0 +1,34 @@
+#include "brakes.hpp"
+
+namespace hyped::sensors {
+
+std::optional<Brakes> Brakes::create(core::ILogger &logger,
+                                     std::shared_ptr<io::IGpio> gpio,
+                                     const std::uint8_t new_pin)
+{
+  const auto reader = gpio->getReader(new_pin, io::Edge::kNone);
```

Comment:
`getWriter` right? or am I misunderstanding and this PR is for monitoring if the brakes are deployed?